tests/hash-backend-integration/README.md

```diff
@@ -2,13 +2,34 @@
 
 ## Mock Data
 
-Mock data may be inserted by running:
+The [`subgraph`](src/tests/subgraph) module contains two folders, which contains snapshots of the Graph, `pass` and `fail`. The Integration test suite is able to restore these snapshots to the Graph, and run the tests against them. In most cases, this requires a clean Graph, so the test suite will create a new Graph, and restore the snapshot to it. For this reason every test should ensure, that the Graph is cleaned up after the test is run.
```

Comment:
```suggestion
The [`subgraph`](src/tests/subgraph) module contains two subdirectories, `pass` and `fail`, each of which houses snapshots of the graph. The integration test suite is able to restore these snapshots to the graph, and run the tests against them. In most cases, this requires a clean graph, so the test suite will create a new graph, and restore the snapshot to it. Because of this, every test should ensure that the graph is cleaned up after it has run.
```